.sbtopts

```diff
@@ -0,0 +1 @@
+-J-Xss2M
```

Comment:
Is this necessary?

Reply:
sbt was OOM: Metaspace-ing on me when running `+test`, but now that I think about it increasing stack size was placebo (solves different sbt issues though)... will revert this one

Reply:
Removed